src/main/java/com/board/tddboard/dto/BoardDto.java

```diff
@@ -0,0 +1,51 @@
+package com.board.tddboard.dto;
+
+import com.board.tddboard.domain.Board;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+
+public class BoardDto {
+
+    @Getter @Setter
```

Comment:
```suggestion
    @Value
    @AllArgsConstructor(access = AccessLevel.PRIVATE)
```

`@Value`를 사용하여 final 필드로 만들어주고
생성자를 만들 수있는 접근제어를 제한해서 불변객체로 만들어주는거 어떤가요?